LSPS5/README.md

```diff
@@ -0,0 +1,512 @@
+
+* For HTTPS webhooks (i.e. `POST`):
+  * The LSP MUST consider a `200 OK` response a
+    success, and MUST ignore the rest of the response,
+    including HTTP headers and the response body.
+  * If the response is not `200 OK`, the LSP SHOULD
+    log this as an unusual event, including the
+    HTTP headers and response body if any, and
+    MUST otherwise ignore the response.
+  * The LSP SHOULD NOT follow any redirection
+    responses.
+
+> **Rationale** As a notification, the user of the
+> client may ignore it, or disable notifications from
+> the client application entirely; thus, delivery of
+> the notification is never assured and it is pointless
+> to feed back anything to the LSP about whether the
+> notification was delivered to the notification
+> delivery service, the mobile OS server, or the client
+> mobile device.
+
+If the notification delivery service accepts the push
+notification, it MUST construct a suitable message to
+display to the user, and send that message, together
+with any required authentication and authorization, to
+the mobile OS server, for display as a push notification
+of the client.
+
+The client SHOULD process what is needed to
+react to the notification (e.g. connect to the LSP and
+accept the payment for a `lsps5.payment_incoming`
+notification) if the mobile environment allows it.
+
+#### Webhook Notification Methods
+
+The LSP MUST use one of the methods listed in this
+section for the webhook notification object.
+
+The LSP MUST NOT use any of these methods on the LSPS0
+interface; these notification methods are only for the
+webhook interface.
+
+Parameters are required unless otherwise stated.
+
+* `lsps5.webhook_registered` - The client has just
+  recently successfully called the `lsps5.set_webhook`
+  API.
+  No parameters `{}`.
+* `lsps5.payment_incoming` - The client has one or
+  more payments pending to be received.
+  No parameters `{}`.
+* `lsps5.expiry_soon` - There is an HTLC or
+  other time-bound contract, in either direction, on
+  one of the channels between the client and the LSP,
+  and it is within 24 blocks of being timed out, and
+  the timeout would cause a channel closure.
+  Parameters:
+  * `timeout` - The blockheight at which the LSP would
+    be forced to close the channel in order to enforce
+    the HTLC or other time-bound contract.
+* `lsps5.liquidity_management_request` - The LSP wants
+  to take back some of the liquidity it has towards the
+  client, for example by closing one or more of the
+  channels it has with the client, or by splicing out.
+  No parameters `{}`.
+* `lsps5.fees_change_incoming` - The LSP wants to change
+  feerates, either for the LSP-to-client channel(s),
+  or for other auto-management services.
+  Parameters:
+  * `direction` - A rough estimate of the direction
+    of the fees, one of the following strings:
+    `"lower"`, `"mixed"`, `"higher"`.
+* `lsps5.goodbye` - The LSP wants to end all services to
```

Comment:
This implies that the LSP will not just close all channels, it will now start rejecting any request from this client, possibly even mere connection. Maybe there is no need for a notification though?

Reply:
I think `liquidity_management_request` is enough and `goodbye` is not needed.